docs/guides/faq.rst

```diff
@@ -0,0 +1,34 @@
+Frequently Asked Questions
+==========================
+
+How do I increase logging level of the testing framework?
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+You can put logging entry into your env.local.yaml like this:
+
+.. code-block:: yaml
+
+    logging:
+        level: DEBUG
+
+Can I run tests in interactive mode?
+""""""""""""""""""""""""""""""""""""
+
+Yes, you can. Just call IPython's method ``embed()`` wherever you need the execution
+to enter an interactive mode. Example:
+
+.. code-block:: python
+
+    def test_foo(bar):
+        x = do_something()
+        from IPython import embed; embed()
+        assert x
+
+Then you can run your test with -s parameter: ``pytest -s cfme/tests/test_foo.py::test_foo``.
+Once the execution reaches the "breakpoint", you will be presented with IPython's
+interactive prompt.
```

Comment:
**OPTIONAL**
IPython is good, but I would add `pdb` as well. Because IPython can't control the execution of a script.

Reply:
Yes I usually use

```import pdb; pdb.set_trace()```